gateway-js/src/datasources/RemoteGraphQLDataSource.ts

```diff
@@ -206,6 +205,23 @@ export class RemoteGraphQLDataSource<
     }
   }
 
+  public makeRequest(
```

Comment:
`makeRequest` sounds like this function executes the request. Maybe `createRequest` or `setupRequest` might be a better name here.

Reply:
@Cretezy setupRequest sound better. If we all agree, I can change this.